### ISSUE TITLE
[Accessibility Issue] Label is not associated with "Check box" in   jsTree - v3.3.7

### ISSUE THREAD

**Issue by Raunak0786** [vakata/jstree#2334]
**Problem**  Label is not associated with "Check box" in   jsTree.
**Repro Steps**:
1. Open any Jstree - v3.3.7

2. Navigate towards checkbox
3. Verify below scenario
a)Screen reader not inform the state and name of the checkbox

**Expected Result**:
a) Label should be associated with "Checkbox"
b) Screen reader Should read it as "(checkbox label name) Checkbox Selected"

**Comment by vakata** [comment]
jstree checkboxes are not real checkbox elements, currently I have no idea on how to achieve this. Feel free to open a PR if you do.
